lib/logstash/inputs/elasticsearch.rb

```diff
@@ -351,6 +343,57 @@ def hosts_default?(hosts)
     hosts.nil? || ( hosts.is_a?(Array) && hosts.empty? )
   end
 
+  def validate_authentication
```

Comment:
same note as in https://github.com/logstash-plugins/logstash-filter-elasticsearch/pull/132 about surprise side-effects from a method with this name

Reply:
As I said, this is also the same in the ES output plugin and I would prefer to keep them aligned. I can change it in both PRs and then submit a patch in the ES output or we defer.

Reply:
+1 to consistency. Ok with resolving in this PR or in a separate effort.

Reply:
I added it back!